app/src/main/java/com/mk/tasky/agenda/detail/presentation/DetailViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.mk.tasky.agenda.detail.presentation
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class DetailViewModel @Inject constructor(
```

Comment:
I'd be a bit more specific here, like `AgendaDetailViewModel` if you'd like to reuse one or `EventDetailViewModel`, `TaskDetailViewModel` etc. if not

Reply:
Thanks
Fixed!